lib/railway_ipc/publisher.ex

```diff
@@ -119,6 +126,7 @@ defmodule RailwayIpc.Publisher do
 
         message
         |> Map.put(:reply_to, callback_queue)
+        |> RailwayIpc.Publisher.ensure_uuid()
```

Comment:
I'm honestly not sure about this one.
`publish_sync` isn't called anywhere in the code, it's only referenced in in the [Learn Docs](https://docs.learn.co/projects/learn-ipc/railway-ipc-hex-package/rpc/rpc_publisher/#call-publish-sync-1) and it expects a protobuff. It then seems to call `publish/1` above which doesn't currently have any tests. `publish/1` though appears to only accept a message of `RailwayIpc.Persistence.PublishedMessage` and it expects this to already have an encoded messages. I'm adding `ensure_uuid/1` really as a guess, but I kind of don't think this works at all and I'm not trying to tackle that in this 'quick' PR.

Reply:
I don't think we're using rpc from the elixir side right now.

Reply:
I ended up pulling this. @Lukeghenco pointed out that the publish here isn't calling the module publish/1 but the meta programmed publish/1 which already has the ensure.